projects/angular-jwt/src/lib/jwt.interceptor.ts

```diff
@@ -40,32 +40,38 @@ export class JwtInterceptor implements HttpInterceptor {
 
   isWhitelistedDomain(request: HttpRequest<any>): boolean {
     const requestUrl: any = parse(request.url, false, true);
-
+    console.log(requestUrl);
     return (
       requestUrl.host === null ||
```

Comment:
If we're checking `hostname` below, does this null check here also need to change?